Makefile

```diff
@@ -22,7 +22,7 @@ run:
 
 .PHONY: fmt
 fmt:
-	go fmt ./...
+	gofmt -s -w .
```

Comment:
`-w` silences the output right? It's useful for us to know which files are modified when running this.